src/ixdat/techniques/ec_ms.py

```diff
@@ -72,14 +72,16 @@ def as_cv(self):
 
         return ecms_cv
 
-    def ecms_calibration(self, mol, mass, n_el, tspan, tspan_bg=None):
+    def ecms_calibration(self, mol, mass, n_el, tspan, isotope_factor=1, tspan_bg=None):
         """Calibrate for mol and mass based on one period of steady electrolysis
 
         Args:
             mol (str): Name of the molecule to calibrate
             mass (str): Name of the mass at which to calibrate
             n_el (str): Number of electrons passed per molecule produced (remember the
                 sign! e.g. +4 for O2 by OER and -2 for H2 by HER)
+            isotope_factor (str): Factor to multiply EC flux by to deconvolute
```

Comment:
Could we just call this faradaic_efficiency? 
With a comment in this docstring that it also works for isotopes.